.gitignore

```diff
@@ -0,0 +1 @@
+node_module
```

Comment:
I think you should add all the unused files here. This makes your documentation professional.

Reply:
[ **OPTIONAL** ] I think it would be better when making a review to use a couple of emojis and/or gifs in the review * They make the code review more human-friendly 👽🤖 and use this semantic below.
> First state the aimed problem ❗, politely offer a suggestion 😏 and conclude with a reason or importance of making that change. uses a couple of emojis and/or gifs in the review.